### PR TITLE
Add advisories for (abandoned?) projects with unfixed vulnerabilities.

### PR DESCRIPTION
**Pull request by paragonie-scott**
* https://github.com/Cosmicist/AsymmetriCrypt/issues/4
* https://github.com/pagarme/pagarme-php/issues/29
* https://github.com/stormpath/stormpath-sdk-php/issues/132
* https://github.com/titon/framework/issues/93

### CODE REVIEW

File: asymmetricrypt/asymmetricrypt/2017-11-20.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:      Padding Oracle Vulnerability in RSA Encryption
+link:       https://github.com/Cosmicist/AsymmetriCrypt/issues/4
+cve:        ~
+branches:
+    0.x:
+        time:     2017-01-19 13:03:00
```
Comment (fabpot) [inline asymmetricrypt/asymmetricrypt/2017-11-20.yaml]: Time is the time when the package was fixed. Not sure what to do here as there is no fix AFAIU. We need to have a new convention. I would suggest to keep the time empty.

Comment (marcomenzel) [reply asymmetricrypt/asymmetricrypt/2017-11-20.yaml]: Propably `~` should be used. 
I just saw it a few times for missing cve-identifiers (like here).

Comment (paragonie-scott) [reply asymmetricrypt/asymmetricrypt/2017-11-20.yaml]: Okay, I'll send a superseding commit right now, then.

Comment (paragonie-scott) [reply asymmetricrypt/asymmetricrypt/2017-11-20.yaml]: Looks like it's failing because `time` is required. Hmm.

Comment (fabpot) [reply asymmetricrypt/asymmetricrypt/2017-11-20.yaml]: Indeed, it's not supported yet by the validator and the server. So, I will need to make some adjustments before merging this PR.

Comment (paragonie-scott) [reply asymmetricrypt/asymmetricrypt/2017-11-20.yaml]: What if we relax that constraint but only if another key is set (e.g. `unfixed: TRUE`)

**Comment by paragonie-scott** [comment]
If #242 is accepted, this should work now. :)

**Comment by paragonie-scott** [comment]
This should be good to go, then.

**Comment by fabpot** [comment]
Indeed, thanks for the pull request.

**Pull request merged**